src/components/TokenMigration/components/TokenMigrationForm.tsx

```diff
@@ -162,6 +170,7 @@ export const TokenMigrationFormUI = ({
 
   return (
     <>
+      {isDemoMode && <DemoGatewayBanner copy={translateRaw('DEMO_GATEWAY_BANNER')} />}
```

Comment:
I wonder if there is a better way to do this than actually inserting this in almost every form 🤔

Reply:
Only thing i can think of is putting it at the top of the stepper component. But we have multiple stepper components at this point still, so it'd still need to be in ~3 places and refactoring that is out of the scope of this pr i think.